ykllvmwrap/src/ykllvmwrap.cc

```diff
@@ -287,19 +287,21 @@ extern "C" void *__ykllvmwrap_irtrace_compile(char *FuncNames[], size_t BBs[],
   llvm::IRBuilder<> Builder(JITContext);
   Builder.SetInsertPoint(DstBB);
 
-  llvm::ValueToValueMapTy VMap;
+  std::vector<ValueToValueMapTy*> vmaps;
+  vmaps.push_back(new llvm::ValueToValueMapTy());
```

Comment:
I think you said that we were unable to allocate the VMaps directly into the vector. Is that still the case?

Reply:
Ah, yes, I forgot to mention this as number 3 why I don't like this approach. We need to allocate the VMaps on the heap. Though, again, that could just be me being bad at C++. ;-) But I couldn't find another way.

Reply:
It did not.